html/tests/spinners.tests.js

```diff
@@ -178,9 +205,8 @@ describe('setSpinning tests', () => {
 
   it('should replace the content with a spinner', () => {
     setSpinning(spinnerContainer, {});
-    expect(
-      spinnerContainer.firstChild.classList.contains('sprk-c-Spinner'),
-    ).toBe(true);
+    const spinnerDiv = spinnerContainer.querySelector('div');
```

Comment:
How come we needed to update this test?

Reply:
It felt more accurate that we were looking for a div with the correct class, rather than any first child that has the correct class